cluster-autoscaler/simulator/cluster.go

```diff
@@ -257,3 +272,10 @@ func shuffleNodes(nodes []*apiv1.Node) []*apiv1.Node {
 	}
 	return result
 }
+
+func checkAnnotations(node *apiv1.Node) error {
```

Comment:
The name and return type is wrong here. Should be called as 
hasNoScaleDownAnnotation (or something like that) and return bool.